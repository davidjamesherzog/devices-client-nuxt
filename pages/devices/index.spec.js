import { createLocalVue, shallowMount } from '@vue/test-utils';
import Buefy from 'buefy';
import Devices from './index.vue';

describe('Devices', () => {
  describe('template', () => {
    let localVue;

    beforeEach(() => {
      localVue = createLocalVue();
      localVue.use(Buefy);
    });

    test('is a Vue instance', () => {
      const wrapper = shallowMount(Devices, { localVue });
      expect(wrapper.vm).toBeTruthy();
      expect(wrapper).toMatchSnapshot();
    });
  });
});
