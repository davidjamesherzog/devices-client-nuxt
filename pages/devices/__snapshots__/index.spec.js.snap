// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Devices template is a Vue instance 1`] = `
<section class="section">
  <h2 class="title is-3 has-text-grey">Devices</h2>
</section>
`;
